data/virt_autotest/host_unattended_installation_files/autoyast_opensuse_kvm_one_disk.xml

```diff
@@ -128,7 +128,8 @@
   <scripts>
     <!-- permit root login, password login and pubkeys login -->
     <!-- keep ssh session alive -->
-    <!-- /etc/ssh/sshd_config is not default installed in TW -->
```

Comment:
Why remove this line? Sshd is installed by default now?

Reply:
this line is a piece of comment. It should have been replaced with the two lines above it in my previous PR, but I forgot it. So It was removed in this PR.

Reply:
Yes so i added in this PR, these were already updated in the xml on ariel